File: python/26_删除有序数组中的重复项_easy.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        n, j = len(nums), 1
        if n <= 1:
            return n
        for i in range(1, n):
            if nums[i] != nums[j-1]:
                nums[j] = nums[i]
                j += 1
        return j

File: cpp/26_删除有序数组中的重复项_easy.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int n = nums.size();
        int j = 1;
        if (n <= 1) return n;
        for (int i=1; i<n; i++)
            if (nums[i] != nums[j-1])
                nums[j++] = nums[i];
        return j;
    }
};